Add a logic to deactivate the plugin when Woo is not active (4780)

Narek13 opened the pull request:
## Issue Description

When WooCommerce plugin is disabled but PayPal Payments is a active, the site crashes with message sometimes.

```
Fatal error: Uncaught LogicException: No PPCP container, probably called too early when the plugin is not initialized yet.
```

###Steps To Reproduce

- set up Multisite
- install WooCommerce & PayPal Payments
- network activate WooCommerce
- navigate to the main site and activate PayPal Payments on the site
- network deactivate WooCommerce
- navigate to main site
- → observe fatal error

## PR Description

Added a logic to deactivate the plugin when Woo is not active

Narek13 committed:
Create a function to use for deactivation with a notice

Narek13 committed:
Deactivate when Woo isn't active or if PHP is lower than required

Narek13 committed:
Deactivate when Woo isn't active or if PHP is lower than required

Narek13 committed:
Do not run any action when Woo isn't active

InpsydeNiklas commented on woocommerce-paypal-payments.php:
```diff
@@ -43,29 +43,84 @@
+						sprintf(
+						// translators: %s is a link to WooCommerce.com.
+							esc_html__( 'WooCommerce PayPal Payments requires WooCommerce to be installed and active. You can download %s here.', 'woocommerce-paypal-payments' ),
+							'<a href="https://woocommerce.com/" target="_blank">WooCommerce</a>'
```
I suggest `network_admin_url('plugin-install.php?tab=plugin-information&plugin=woocommerce')` as target instead of  `woocommerce.com`. 
Additionally, I suggest including the entire sentence `You can download WooCommerce here.` in the linked text as well.

Narek13 committed:
Improve WooCommerce dependency notice

- Link to WordPress plugin install page instead of external site
- Make entire call-to-action sentence clickable
